.gitignore

```diff
@@ -6,5 +6,8 @@
 /docs/build/
 .DS_Store
 
+.idea
+
+test/figures
```

Comment:
@Omadzze maybe actually remove them from gitignore. It would be nice to have those figures in git, so that we can  use them in the docs.